frontend/src/components/editCard/ModifyEdit.tsx

```diff
@@ -190,20 +198,20 @@ const ModifyEdit: React.FC<ModifyEditProps> = ({
         <ChangeRow
           name="Tattoos"
           newValue={(details?.added_tattoos ?? [])
-            .map((tatt) => `${tatt.location}: ${tatt.description}`)
+            .map(formatBodyModification)
             .join("\n")}
```

Comment:
The newline does not show up visibally on production.
I assume you don't want HTML here, so maybe `white-space: pre-wrap;` will fix it?

Reply:
True, added it in. Thanks!